src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -81,6 +81,29 @@ def _any_sku_values_specified(sku):
     return any(val for key, val in sku.__dict__.items())
 
 
+def _get_default_server_version(location_capabilities):
+    '''
+    Gets the default server version capability from the full location
+    capabilities response.
+
+    If none have 'default' status, gets the first capability that has
+    'available' status.
+
+    If there is no default or available server version, falls back to
+    server version 12.0 in order to maintain compatibility with older
+    Azure CLI releases.
+    '''
+    server_versions = location_capabilities.supported_server_versions
+
+    try:
+        # Try behavior from azure-cli-sql 2.0.26: get default
+        return _get_default_capability(server_versions)
+    except StopIteration:
+        # No default or available version found.
+        # Fall back to behaviour from azure-cli-sql 2.0.25 and earlier: get version 12.0
+        return next(sv for sv in server_versions if sv.name == "12.0")
```

Comment:
You might issue a warning in this case if it's abnormal.

Reply:
How would I do that?

Reply:
logger.warning(…)